<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
	<head>
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
		<script  type="text/javascript">
		$(document).ready(function(){
			$('#page #navigator li a').click(function(e) {
				$('#page #navigator li a.active').removeClass('active');
				var $this = $(this);
				if (!$this.hasClass('active')) {
					$this.addClass('active');
				}
				e.preventDefault();
				
				var href=$(this).attr('href');
				$("html, body").animate({ scrollTop: $(href).offset().top });

			});
		});
		</script>			
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
		<title> My personal homepage </title>
		<link href="styles/default.css" rel="stylesheet" type="text/css" media="screen"/>
	</head>
	<body>
		<div id="container"> 
			<div id="header">			
				<div id="faceshot">
					<img src="images/FaceShot.png" alt="Photo of <b>Shahin Mohammadi</b>" width="200"/>
				</div>
				<div id="affiliation">
					<h1>Shahin Mohammadi</h1>
					<p style="margin-bottom:14pt;">
						Ph.D. Candidate</br>
					    Computational Biologist
					</p>								
				</div>	
				<div style="float: right; padding-right:18pt;">															
					<a href="mailto:mohammadi@purdue.edu" title="E-mail"><img src="images/Email.png" width=32pt ></a>					
					<a href="https://www.linkedin.com/profile/view?id=158845072" title="LinkedIn"><img src="images/LinkedIn.png" width=32pt ></a>
					<a href="https://www.facebook.com/shahin.mohammadi.10" title="Facebook"><img src="images/Facebook.png" width=32pt ></a>
					<a href="https://github.com/shmohammadi86" title="GitHub"><img src="images/GitHub.png" width=32pt ></a>						
				</div>				
				<div style="clear:both; display:block;"></div>						
			</div> <!-- header -->

			<div id="menu">  <!-- TODO: Add spacing between menu items -->	
				<ul class="flex-container">
					<li class="flex-item"><a href="index.html" title="">Home</a></li>
					<li class="flex-item"><a href="bio_page.html" title="">Bio& CV</a></li>
					<li class="flex-item"><a href="summary_page.html" title="">Research Summary</a></li>
					<li class="flex-item"><a class="active" href="projects_page.html" title="">Projects</a></li>
					<li class="flex-item"><a href="gallery_page.html" title="">Gallery</a></li>
					<li class="flex-item"><a href="contact_page.html" title="">Contact Me</a></li>
				</ul>
			</div> <!-- menu -->	

			<div id="page">
				<div id="navigator" style="width:20%;">
					<ul style="font-size:14pt;">
					  <li><a class="active" href="#motif">Network motif detection</a></li>
					  <li><a href="#alignment">Network alignment</a></li>
					  <li><a href="#aging">Constructing effective response network of calorie restriction</a></li>
					  <li><a href="#ts_net">Multi -omics reconstruction of tissue-specific interactomes</a></li>					  
					  <li><a href="#hs_vs_sc">Comparative analysis of tissue-specific networks</a></li>
					  <li><a href="#marker">Marker detection</a></li>
					  <li><a href="#deconvolution">Insilico deconvolution</a></li>					  
					</ul>					
				</div>
				
				<div id="content" style="margin-left:25%; width:75%">
					<h1>Research Areas and Projects</h1>
					<h2 id="motif">Network motif detection</h2>	
						<p>Complex networks are studied across many fields of science and are particularly important to understand biological processes from a systems point of view. The existence of specific sub-graphs that repeat themselves within and across networks would be consistent with the tenets of evolutionary theory. Each of these sub-graphs, defined by a particular pattern of interactions between vertices, may reflect a framework in which particular cellular functions are performed efficiently. These sub-graphs are called network motifs. Motifs are of notable importance largely because they reflect conserved functional properties of biological networks. Nevertheless, as possible associated functions may be unknown initially, defining motifs independent of the function and based on the frequency of their occurrence is commonly accepted as an alternative method. As such, motifs can be considered as specific subgraphs, which occur at significantly higher frequencies in the real network compared to an ensemble of random networks with similar topological properties.</p>
						<p>Unfortunately, the task of discovering network motifs, also known as motif-finding problem, is extremely demanding both in terms of the processing power and memory needs. During this project, we have developed a novel method, called Kavosh, which enumerates all k-size sub-graphs of a given graph (directed or undirected), and is the only method that is capable of counting motifs of size greater than 8. At the core of our method is an extremely efficient enumeration method that expands the neighborhood of each node in a systematic way to optimize resource usage and minimize computational redundancy. This method is then coupled with a statistical framework to assess the significance of each observed structure and report ones that are statistically significant, when compared to an ensemble of random graphs with same degree distribution as the original network. Our method allows us to identify larger network motifs that were never possible before with traditional methods, and greatly enhances our understanding of the underlying structures in complex networks.</p>						
						<div style="padding-left:30pt">
						<h3>Relevant publications:</h3>
						<ol>
							<li>Z. R. M. Kashani, H. Ahrabian, E. Elahi, A. Nowzari-Dalini, E. S. Ansari, S. Asadi, <b>S. Mohammadi</b>,
							F. Schreiber, and A. Masoudi-Nejad (2009) <a href="http://dx.doi.org/10.1186/1471-2105-10-318">Kavosh: a new algorithm for finding network motifs</a>.
							BMC bioinformatics, 10(318)</li>
						</ol>
						</div>

									
					<h2 id="alignment">Comparative analysis of complex networks</h2>					
						<p>Many of the problems in network biology originate from their counterpart problems in sequence analysis. The concept of network motifs is an extension of sequence motifs, short, recurring, patterns in DNA sequence that are believed to have conserved functions. Similarly, network alignment is a natural extension of the sequence alignment to transfer knowledge across species. Network alignment, or finding an optimal mapping between a set of given networks, is a fundamental problem that can provide invaluable insights on the similarities and differences of input networks, and transfer knowledge across domains. However, when defined on general graphs, this problem is considerably harder than its well-studied counterparts on sets and sequences. Even in the simplest cases, this problem is usually intractable to solve exactly, as it can be proven to be NP-Hard, and efficient methods are needed to approximate their solutions.</p>
						<p>Our research expands the state-of-art of network alignment in two main aspects. First, we have developed various implementations to speed-up the computational time of network alignment by an order of magnitude and allow us to apply it to networks with thousands or even millions of nodes. Next, we have devised new formulations to enhance the overall quality of computed alignments by incorporating higher-order structures and prior knowledge of biological pathways in the objective function of the network alignment. Finally, we have been developing highly sophisticated tensor-based methods to combine the network motif and network alignment problems under the same framework to directly align network-motifs across networks.</p>
						<div style="padding-left:30pt">
						<h3>Relevant publications:</h3>
						<ol>
							<li><b>S. Mohammadi</b>, D. Gleich, T. G. Kolda, and A. Grama (2015) Triangular AlignMEnt (TAME): A
							novel tensor-based approach for global network alignment. (manuscript under preparation for
							submission to Bioinformatics)</li>							
							<li>G. Kollias, M. Sathe, <b>S. Mohammadi</b>, and A. Grama (2013) <a href="http://dx.doi.org/10.1186/1756-0500-6-35">A fast approach to global alignment
							of protein-protein interaction networks</a>. BMC research notes, 6(1):35</li>
							<li>G. Kollias, <b>S. Mohammadi</b>, and A. Grama (2011) <a href="http://dx.doi.org/10.1109/TKDE.2011.174">Network Similarity Decomposition (NSD): A
							Fast and Scalable Approach to Network Alignment</a>. IEEE Transactions on Knowledge and Data
							Engineering (TKDE), 24(12):2232–2243</li>
							<li><b>S. Mohammadi</b> and A. Grama (2011) <a href="http://dx.doi.org/10.1007/978-1-4614-0320-3">Biological Network Alignment</a>. In M. Koyuturk, S. Subramaniam, and A. Grama (eds.), Functional Coherence of Molecular Networks in Bioinformatics, Springer, pages 97–136</li>
						</ol>
						</div>


					<h2 id="aging">Computational map of aging pathways: Uncovering downstream effectors of calorie restriction in yeast</h2>
						<p>Cellular aging is a multifactorial complex phenotype, characterized by the accumulation of damaged cellular components over the organism’s life-span. The progression of aging depends on both the increasing rate of damage to DNA, RNA, proteins, and cellular organelles, as well as the gradual decline of cellular defense mechanisms against stress. This can ultimately lead to a dysfunctional cell with a higher risk factor for disease. The traditional methodology for elucidating the underlying mechanisms of lifespan mediation relies on genetic alteration of single genes (and consequently their products), either in the form of null mutant or over/under-expression of the gene, and assessing its impact on longevity. Different molecular mechanisms, including decreasing telomere integrity, ERC accumulation, increasing level of oxidative damage, and genomic instability, have been proposed to explain the various processes that mediate lifespan. However, owing to the multifactorial nature of aging, it is unlikely that any of these mechanisms mediate lifespan in isolation, since each one of them only accounts for a specific aspect of the cellular machinery.</p>
						<p>Calorie restriction (CR) is one of the most conserved non-genetic interventions, which extends healthy lifespan in evolutionarily distant species ranging from yeast to mammals. Yeast cells are typically cultured in growth media containing 2% glucose. Reducing glucose concentration to 0.5% or less is one of the best characterized CR regimens in yeast that increases both replicative and chronological lifespan. The target of rapamycin (TOR) has been shown to play a key role in mediating health-span extension in response to CR by integrating different signals that monitor nutrient-availability and orchestrating various components of cellular machinery in response. In addition, both genetic and pharmacological interventions that inhibit the TOR pathway exhibit a similar phenotype, which is not further amplified by CR. As such, inhibiting TOR signaling using Rapamycin provides a unique opportunity to identify CR-mediated downstream effectors. Recent advances in high-throughput technologies provide us with cellular blueprints of yeast at different levels of complexity including genome, transcriptome, proteome, and metabolome. These datasets provide an invaluable asset for deciphering the system-level organization of underlying pathways. However, rapid growth and inherent noise in these datasets make it particularly challenging to develop reliable models and methods to integrate these datasets and facilitate the inference and reconstruction of aging pathways.</p>
						<p>Our long-term goal is to develop models and methods aimed at a mechanistic understanding of the pathways that mediate health-span in model organisms, project these pathways to different human tissues, and assess their significance with respect to the onset and progression of age-related pathologies in humans. To this end, we developed a systematic approach to construct an integrated interactome consisting of protein-protein, gene regulatory, and signaling interactions in yeast. We used this interactome, and built an information flow analysis method to rank genes with respect to the received signal from TOR pathway. Finally, we developed a novel statistical framework for integrating information flow scores, data on regulatory relationships, and the expression profile in response to rapamycin treatment. Using our framework, we were able to identify the most relevant transcription factors and construct the effective response network of TOR, which is responsible for the observed transcriptional changes due to TOR inhibition.	</p>
						<div style="padding-left:30pt">
						<h3>Relevant publications:</h3>
						<ol>
							<li><b>S. Mohammadi</b>, S. Subramaniam, and A. Grama (2013) <a href="http://dx.doi.org/10.1186/1752-0509-7-84">Inferring the Effective TOR-Dependent
							Network: A Computational Study in Yeast</a>. BMC Systems Biology, 7(1):84</li>
							<li>A. Grama and <b>S. Mohammadi</b> (2013) <a href="pdfs/BigData_presentation.pdf">A Systems Study of Aging and Age-Related Pathologies</a>.
							CSoI Big Data Workshop, Honolulu, HI, USA</li>
							<li><b>S. Mohammadi</b>, S. Subramaniam, and A. Grama (2012) <a href="pdfs/RECOMB_poster.pdf">Systematic Identification of TOR Downstream
							Effectors Using Random-Walks on the Yeast Interactome</a>. RECOMB Conference on Systems
							Biology and Regulatory Genomics, San Francisco, CA, USA</li>
							<li><b>S. Mohammadi</b>, S. Subramaniam, and A. Grama (2012) <a href="pdfs/ISMB_NetBio_poster.pdf">Sweet-talking yeast: systematic identification
							of TOR downstream effectors under dietary restriction</a>. Network Biology SIG, Intelligent
							Systems for Molecular Biology (ISMB), Long Beach, CA, USA</li>							
						</ol>
						</div>						


					<h2 id="ts_net">Construcing the most informative active sub-network in tissues/ cell-types</h2>
						<p>To be written ... </p>
						<div style="padding-left:30pt">
						<h3>Relevant publications:</h3>
						<ol>
						</ol>
						</div>	
						
						
					<h2 id="hs_vs_sc">Scope and limitations of yeast as a model organism for studying human tissue-specific pathways</h2>
						<p>Budding yeast, S. cerevisiae, has been used extensively as a model organism for studying cellular processes in evolutionary distant species, including humans. However, different human tissues, while inheriting a similar genetic code, exhibit distinct anatomical and physiological properties. Specific biochemical processes and associated biomolecules that differentiate various tissues are not completely understood, neither is the extent to which a unicellular organism, such as yeast, can be used to model these processes within each tissue.</p>
						<p>We present a novel framework to systematically quantify the suitability of yeast as a model organism for different human tissues. To this end, we develop a computational method for dissecting the global human interactome into tissue-specific cellular networks. By individually aligning these networks with the yeast interactome, we simultaneously partition the functional space of human genes, and their corresponding pathways, based on their conservation both across species and among different tissues. Finally, we couple our framework with a novel statistical model to assess the conservation of tissue-specific pathways and infer the overall similarity of each tissue with yeast. We further study each of these subspaces in detail, and shed light on their unique biological roles in the human tissues.</p>
						<div style="padding-left:30pt">
						<h3>Relevant publications:</h3>
						<ol>
							<li><b>S. Mohammadi</b>, B. Saberidokht, S. Subramaniam, and A. Grama (2015) <a href="http://dx.doi.org/10.1101/011858">Scope and limitations
							of baker’s yeast as a model organism for studying human tissue-specific pathways</a>. BMC Systems
							Biology (accepted)</li>
							<li><b>s. Mohammadi</b>, B. Saberidokht, S. Subramaniam, A. Grama (2014) <a href="pdfs/NAS_presentation.pdf">A Systems Approach to Dissect the tissue-specific architecture of cellular networks</a>.
							National Academy of Sciences, Washington, DC, USA
							</li>
							<li><b>S. Mohammadi</b>, B. Saberidokht, and A. Grama (2013) <a href="pdfs/GLBio_poster.pdf">Comparative analysis of yeast interactome
							versus human tissue-specific networks</a>. Great Lakes Bioinformatics Conference (GLBIO),
							Pittsburgh, PA, USA</li>							
						</ol>
						</div>	
						
						
					<h2 id="marker">Cell-type markers revisited: using hierarchical priors to identify compound markers as cellular barcodes for cell-type identification</h2>
						<p>Identification of biochemical processes that drive the transformation of a totipotent cell into various cell types is essential to our understanding of living systems. This complex machinery determines how tissues differ in terms of their anatomy, physiology, morphology, and, more importantly, how various cellular control mechanisms contribute to the observed similarities/ differences. Tissue-selective genes orchestrate various aspects of cellular machinery in different tissues, and are known to be implicated in a number of tissue-specific pathologies.</p>
						<p>We propose a novel statistical approach for constructing a reliable network of tissue-tissue similarities. We utilize this network, as a refined prior to identify the underlying grouping of human tissues, and propose new method to uncover tissue-selective functions and their corresponding genes. Using a case study of brain tissues, we show that our approach significantly outperforms state-of-the-art methods, both in terms of coverage and reliability of predicted tissue-selective genes, as well as their significance in characterizing pathophysiology of human tissues. Our results include a first-of-its-kind network of tissue-tissue similarity that provides a means for projecting prior knowledge of tissue-specific functions across the set of related tissues. These results also include highly reliable brain-selective functions and genes, as well as a general framework that can be extended to construct similar datasets in other human tissues. These tissue-selective targets greatly enhance our understanding of differentiating biochemical features of brain tissues, shed light on how brain-selective pathologies progress, and help us identify specific biomarkers and targets for future interventions.</p>
						<div style="padding-left:30pt">
						<h3>Relevant publications:</h3>
						<ol>
							<li><b>S. Mohammadi</b> and A. Grama (2015) <a href="http://dx.doi.org/10.1101/020685">A novel method to enhance the sensitivity of marker detection
							using a refined hierarchical prior of tissue similarities</a>. BMC Bioinformatics (under review)</li>
						</ol>
						</div>							

						
					<h2 id="deconvolution">In-silico deconvolution of mixed/ contaminated expression profiles</h2>
						<p>Collected biological samples are typically heterogeneous in terms of their constituting cell-types, and dynamic changes in the relative cell-type proportions in different conditions can reflect the underlying biological response and is a driving element in progression of disorders ranging from neurodegenerative diseases to cancers. Gradual depletion of specific subsets of neuronal cells accompanied by a gradual increase in the population of glial cells is a hallmark of neurodegenerative diseases, whereas changes in the tumor microenvironment composition, such as increasing proportion of malignant cells or tumor infiltrating lymphocytes (TILs) is highly associated with the clinical outcome of patients. Another common use-case is the relative changes in immune cell subsets i the peripheral blood, which can be used for immune monitoring in the course of an infection. However, the inherent heterogeneity of mixed tissue samples hinders our ability to identify pure, cell-type signatures and any biological variation can be masked by the confounding changes in tissue composition. Consequently, to fully understand gene expression differences associated with different physiological states, deconvolution of tissue expression into the component expression profiles of each cell type is critically needed.</p>
						<p>A variety of experimental cell separation methods have been proposed to enrich samples for cell types of interest. Fluorescence Activated Cell Sorting (FACS) is a flow cytometry-based method that can separate mixed cell-types in the solution (such as blood samples), whereas Laser Capture Microdissection (LCM) has been used to isolate morphologically distinguishable cells in solid tissues before gene expression analysis. The limited availability of high-quality, specific surface markers as well as the introduced cellular stress during the sorting process hinders the application of these methods. More recently, Translating Ribosome Affinity Purification (TRAP) have been proposed to partially remedy these problems. However, these methods are not only time-consuming, costly, and labor intensive, but also they may result in insufficient quantities of RNA, where amplification steps may introduce technical artifacts to the gene expression data.</p>
						<p>To address these issues, we are collaborating with Dr. Goldsmith’s lab at Stanford to build cutting-edge algorithms for in silico deconvolution of mixed samples into their constituent cell-type specific expression profiles and their relative proportions. Our research focuses on two main aspects of the deconvolution problem that have been largely unexplored so far. First, we are building a real-time deconvolution method, in which a single-sample from a patient can be directly deconvolved. This formulation is ideal for use in clinical applications in which each patient’s profile needs to be processed individually as they come in. Second, we are focusing on deep-deconvolution, in which samples with high number of cell-types with varying degrees of similarity can be deconvoluted. This would be particularly useful for purifying tumor samples, as biopsy samples usually contain wide range of cell-types ranging from immune cell, fibroblast, endothelial, as well as a heterogenous population of tumor subtypes.</p>
						<div style="padding-left:30pt">
						<h3>Relevant publications:</h3>
						<ol>
							<li><b>S. Mohammadi</b>, N. Zuckerman, A. Golsmith, and A. Grama (2015) Comprehensive review on the
							choice of loss function, regularization, and data preprocessing on the performance of deconvolution
							methods. (manuscript under preparation for submission to the Special Issue of Proceedings
							of IEEE - Foundations & Applications of Science of Information)</li>
						</ol>
						</div>	
				</div>											
			</div> <!-- page -->				
		</div> <!-- container -->	
	</body>
</html>
